invenio_rdm_migrator/transform/base.py

```diff
@@ -23,15 +29,35 @@ def _transform(self, entry):  # pragma: no cover
         """
         pass
 
-    def run(self, entries):
-        """Transform and yield one element at a time."""
-        for entry in entries:
+    def _multiprocess_transform(self, entries):
+        def _transform(entry):
             try:
-                yield self._transform(entry)
+                return self._transform(entry)
             except Exception:
                 logger = Logger.get_logger()
                 logger.exception(entry, exc_info=1)
-                continue
+
+        for result in pypeln.process.map(
```

Comment:
🚀

Reply:
That was a bit of a failed experiment to try speeding things up for the transform step (which is a textbook "embarrassingly parallel" task)... Apparently, there's so much overhead for passing dictionaries across processes (i.e. IPC) that it run even slower... I gave up after a couple of tweaks, but it might be worth exploring to improve on snapshot loading time.

Reply:
Was the SQLite very slow on passing around also? could this compensate its slowness? wondering if DB+pypeln might be better than dict and sequential.